doc/source/cluster/kuberay/kuberay-vs-legacy.md

```diff
@@ -0,0 +1,24 @@
+(kuberay-vs-legacy)=
+
+# KubeRay vs. the legacy Ray Operator
```

Comment:
There's naturally been a lot of user concern on this topic.
Definitely something we need to elucidate.